api/v2/helpers/credentials.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import print_function
+import datetime
+import pickle
+import os.path
+import json
+
+from flask import make_response, jsonify
+from googleapiclient.discovery import build
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow, Flow
+from google.auth.transport.requests import Request
+
+from api.v2.models.bouquets.bouquets_model import Bouquets
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ['https://www.googleapis.com/auth/calendar.readonly']
+
+
+def calendar(data):
```

Comment:
This is a bad function name. Function names should have descriptive verbs. This looks like a variable name.